metadata_tools/columns.py

```
@@ -18,12 +18,11 @@ def get_bodies(body_names):
     return {body.name: body for body in bodies}
 
-
 BODIES = get_bodies(defs.BODY_NAMES)
 
 #############################################
 # Define geometry columns
 #############################################
 column_files = list(defs.COLUMN_DIR.glob('COLUMNS_*.py'))
 for file in column_files:
-    exec(open(file).read())
+    exec(open(file.as_posix()).read())
```

Ruff (0.14.10)

Use of `exec` detected

(S102)